modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/ObjectMapperUtil.java

```diff
@@ -32,6 +32,20 @@
  */
 public class ObjectMapperUtil {
 
+	public static <T> T readValue(Class<?> clazz, Object element) {
+		try {
+			return (T)_objectMapper.readValue(
```

Comment:
Can we replace this code calling the other readValue method with _objectMapper.writeValueAsString(element) as argument?

Reply:
Thinking about this. I think there is another method in the ObjectMapper that may fit better to this use case that is [convertValue](https://fasterxml.github.io/jackson-databind/javadoc/2.8/com/fasterxml/jackson/databind/ObjectMapper.html#convertValue(java.lang.Object,%20java.lang.Class))